pkg/kubelet/kubelet_pods.go

```diff
@@ -206,7 +206,6 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 				VolumePath:       volumePath,
 				PodDir:           podDir,
 				ContainerName:    container.Name,
-				ReadOnly:         mount.ReadOnly || vol.Mounter.GetAttributes().ReadOnly,
```

Comment:
why was this removed?

Reply:
@liggit by mistake, not sure when and how this got removed. Updated.